src/Symfony/Component/HttpClient/Tests/CurlHttpClientTest.php

```diff
@@ -120,4 +121,24 @@ public function testOverridingInternalAttributesUsingCurlOptions()
             ],
         ]);
     }
+
+    /**
+     * @testWith [1]
+     *           [5]
+     */
+    public function testOverridingConnectTimeoutUsingCurlOptions(int $connectTimeout)
+    {
+        $httpClient = $this->getHttpClient(__FUNCTION__);
+        $this->expectException(TransportException::class);
+        $this->expectExceptionMessageMatches(\sprintf('~Connection (timeout|timed out) after %d (ms|milliseconds) for "http\://10\.255\.255\.1/"\.~', $connectTimeout));
```

Comment:
The message I got locally is slightly different than the one on the CI, I'm guessing it depends on the version of Curl.